src/couch_replicator_manager.erl

```diff
@@ -213,11 +216,13 @@ handle_cast(Msg, State) ->
     twig:log(error, "Replication manager received unexpected cast ~p", [Msg]),
     {stop, {error, {unexpected_cast, Msg}}, State}.
 
-handle_info({nodeup, _Node}, State) ->
-    {noreply, rescan(State)};
+handle_info({nodeup, Node}, State) ->
+    Live = lists:usort([Node|State#state.live]),
```

Comment:
further investigation and I'm inclining to agree. The replication manager is terrible, I have other bug fixes coming.